docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -592,30 +600,190 @@ depending on where they wind up.-->
         <para>The package maps page, displays all the package maps loaded on
         your cluster. You can Add, Activate, Deactivate, Delete, or Open a
         package map. To examine a package map, select a package map from the
-        list.</para>
+        list. </para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <para>To update the package maps you are using, you would either edit
+        the package map file (or copy then add) or add a new one. Then
+        activate the new one, and then delete the old one. </para>
 
-        <para><figure>
-            <title>Validate Package Maps</title>
+        <sect3>
+          <title id="Packagemap_Actions">Package Map Actions</title>
 
-            <mediaobject>
-              <imageobject>
-                <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
-              </imageobject>
-            </mediaobject>
-          </figure></para>
+          <para>You can perform actions on your package maps from the Package
+          Maps tab in ECL Watch. </para>
+
+          <para><emphasis role="bold">Package Map Open</emphasis></para>
+
+          <para>Examine your package map closer by selecting the package map,
+          and then press the <emphasis role="bold">Open</emphasis> action
+          button. This opens a new tab where you can see the package parts.
+          </para>
+
+          <para><emphasis role="bold">Package Map Add </emphasis></para>
+
+          <para><figure>
+              <title>Add Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWA470A.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure>To Add a package map to the target cluster:</para>
+
+          <orderedlist>
+            <listitem>
+              <para>Select the package map to add by checking the box next to
+              it.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Add</emphasis> action
+              button and open the Add Package Map dialog.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Select Package
+              File</emphasis> button and select the package files to
+              add.</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose the <emphasis role="bold">Target </emphasis>to
+              associate the package map with.</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose a Process FIlter /*what is command line
+              equiv?*/</para>
+            </listitem>
```

Comment:
You need to remove this comment and describe the field. There is no command line equivalent, so ask how this is used.